src/client/conn/http2.rs

```diff
@@ -426,7 +426,7 @@ where
         let opts = self.clone();
 
         async move {
-            trace!("client handshake HTTP/1");
+            trace!("client handshake HTTP2");
```

Comment:
nit: how about
```suggestion
            trace!("client handshake HTTP/2");
```

Reply:
I don't mind personally.
I picked `HTTP2` because in the doc on the same file, this HTTP version is most often referred to as `HTTP2`.
But there is also 1 mention of `HTTP/2`